### ISSUE TITLE
PyCharm custom config path not supported

### ISSUE THREAD

**Issue by axel-kah** [kiteco/plugins#151]
Hi,

people with a custom config path for PyCharm are having a hard time using Kite. Kite will detect the installation but is failing to install unless you use the default config folder path for that particular version.

I propose you check the installations custom config path first and only fall back to other strategies if this is not successfull.

PyCharms (IntelliJ's) official way to customize paths on Windows is to edit the `idea.config.path` variable, located in `<PyCharmInstallFolder>\bin\idea.properties`.

Cheers,
Axel

**Comment by metalogical** [comment]
Thanks for the bug report.

@jansorg can you confirm this behavior?

**Comment by jansorg** [comment]
@metalogical Yes, I can confirm this. Kite is checking the default locations, but it (not yet)  checking customized locations. One way to customize is to set `idea.config.path`, another way is to set `idea.plugins.path` in the `idea.properties` file. `idea.plugins.path` defaults to `${idea.config.path}/plugins`.

**Comment by its-dhung** [comment]
@jansorg is this fixed with the new running editor detection logic? Feel free to close if so.
